README.md

```diff
@@ -20,37 +21,9 @@
 
 This class manages SELinux on RHEL based systems.
 
----
```

Comment:
We need to persist and update this message with all details of what has been changed to conform to the Apache 2.0 License.

This is the reason that I didn't bother updating the test structure at the last migration.